python/src/worker/config/__init__.py

```diff
@@ -7,15 +7,12 @@
 import redis
 from aws_xray_sdk import core, global_sdk_config
 
+
 kube_env = os.getenv("K8S_ENV")
 cluster_env = os.getenv("CLUSTER_ENV")
-
-timeout = int(
-    os.getenv(
-        "WORK_TIMEOUT",
-        default=str(60 * 10),
-    )
-)
+default_timeout = 60 * 10
+timeout = os.getenv("WORKER_TIMEOUT", default = str(default_timeout))
+timeout = int(timeout) if timeout else default_timeout
```

Comment:
necessary for if `WORKER_TIMOUT` is empty string. `os.getenv(..., default=)` only uses default if variable is `None`